### PR TITLE
added network object support

### PR DESCRIPTION
**Pull request by pruan-rht**
helper for #661

### CODE REVIEW

**Comment by pruan-rht** [comment]
@akostadinov PTAL.  thanks

File: lib/openshift/network.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift environment Storage Class
+  class Network < ClusterResource
+    RESOURCE = "networks"
```
Comment (akostadinov) [inline lib/openshift/network.rb]: usually RESOURCE is `io.openshift...` nowadays. You will see this in YAML output of objects.

Comment (akostadinov) [reply lib/openshift/network.rb]: other than that lg

File: lib/openshift/network.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift environment Storage Class
```
Comment (liangxia) [inline lib/openshift/network.rb]: Comment/Code mismatch

**Pull request closed without merging**